CHANGELOG.md

```diff
@@ -11,7 +11,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   (cyberark/conjur-authn-k8s-client#70)
 
 ### Fixed
-- Fixed issue with providing complex Conjur secrets (#77)
+- Fixed issue with providing complex Conjur secrets (#77). The secrets-provider
```

Comment:
I don't think complex conjur secrets is enough to explain what was done?

maybe add "Fixed issue with providing complex Conjur secrets such as JSON and ssh secrets..."

Reply:
```
- Fixed issue with providing complex Conjur secrets (e.g JSON, SSH keys)
  [cyberark/secrets-provider-for-k8s#77](https://github.com/cyberark/secrets-provider-for-k8s/issues/77). 
  The secrets-provider now updates k8s secrets using `update` instead of `patch` so the service-account
  needs to have that permission.
```

WDYT?